Watchman.Discord/WatchmanBot.cs

```diff
@@ -49,15 +45,9 @@ public WorkflowBuilder GetWorkflowBuilder()
                         .AddHandler(() => Task.Run(() => Log.Information("Bot started and logged in...")))
                         .AddFromIoC<ConfigurationService>(configurationService => configurationService.InitDefaultConfigurations)
```

Comment:
a gdyby tak tego buildera rozbić na kilka metod
które na wejściu mają WorkflowBuilder (referencje)

i wiesz
każda metoda zajmuje sie jakimś zakresem z konfiguracji 
albo nawet podzielić to na osobne klasy gdyby wtedy było to zbyt potężne

bo teraz jest już troche tak że jest wszystko upchnięte w jednym miejscu 
i no niby sam zapis jest czytelny
ale jak masz mase różnych konfiguracji
to trudno coś w tym znaleźć tak wiesz - odruchowo 
musisz wszystko przeszukiwać, jeśli nie znasz tego na pamięć

Reply:
ale to tylko propozycja - może być nawet kiedyś tam w przyszłości